src/ZipFile.jl

```diff
@@ -497,13 +499,13 @@ function position(f::ReadableFile)
 end
 
 # Write nb elements located at p into f.
-function write(f::WritableFile, p::Ptr, nb::Integer)
+function unsafe_write(f::WritableFile, p::Ptr, nb::Integer)
```

Comment:
also, what's up with `b = reinterpret(UInt8, reshape(a, length(a)))`, it looks like a complicated no-op?